packages/veritone-widgets/src/redux/modules/mediaDetails/faceEngineOutput/index.js

```diff
@@ -546,24 +555,26 @@ export const getFaceEngineAssetData = (state, engineId) => {
   const userDetectedFaces = getUserDetectedFaces(state, engineId);
   const userRemovedFaces = getUserRemovedFaces(state, engineId);
 
-  // On the result use engineAliasId for 'user-edited-face-engine-results'
   const userEdited = {
-    sourceEngineId: '7a3d86bf-331d-47e7-b55c-0434ec6fe5fd',
+    sourceEngineId: engineId,
     sourceEngineName: 'User Edited'
   };
 
-  const allSeries = addUserDetectedFaces(
+  const allJsonData = addUserDetectedFaces(
     engineResults,
     userDetectedFaces,
     userRemovedFaces
   ).reduce((accumulator, engineResult) => {
-    return [...accumulator, ...engineResult.series];
+    return [
+      ...accumulator,
+      {
+        ...userEdited,
+        ...engineResult
+      }
+    ];
   }, []);
 
-  return {
-    ...userEdited,
-    series: allSeries
-  };
+  return allJsonData;
```

Comment:
as previously this should collapse all series into one array, preferably sort it, and return single JSON, not array